pysass.cpp

```diff
@@ -404,6 +435,116 @@ static void _add_custom_functions(
     sass_option_set_c_functions(options, fn_list);
 }
 
+Sass_Import_List _call_py_importer_f(
+        const char* path,
+        Sass_Importer_Entry cb,
+        struct Sass_Compiler* comp
+) {
+    PyObject* pyfunc = (PyObject*)sass_importer_get_cookie(cb);
+    PyObject* py_path = PyUnicode_FromString(path);
+    PyObject* py_args = PyTuple_New(1);
+    PyObject* py_result = NULL;
+    Sass_Import_List sass_imports = NULL;
+    Py_ssize_t i;
+    
+    PyTuple_SetItem(py_args, 0, py_path);
+    
+    if (!(py_result = PyObject_CallObject(pyfunc, py_args))) {
```

Comment:
I'm pretty sure they all go through `__call__`:

(at least because this works):
```python
>>> def f(a): print(a)
... 
>>> f.__call__
<method-wrapper '__call__' of function object at 0x7f7419588668>
>>> f.__call__(2)
2
```